.coderabbit.yaml

```
@@ -11,6 +11,8 @@ reviews:
   auto_review:
     enabled: true
     drafts: false
+    base_branch: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)